aws/s3_test.go

```diff
@@ -33,39 +33,67 @@ func TestMain(m *testing.M) {
 	os.Exit(exitVal)
 }
 
-// TestS3Bucket represents a test S3 bucket.
-type TestS3Bucket struct {
-	name        string
-	tags        []map[string]string
-	isVersioned bool
+// genTestsBucketName generates a test bucket name.
+func genTestBucketName() string {
+	return strings.ToLower("cloud-nuke-test-" + util.UniqueID() + util.UniqueID())
 }
 
-// SetupInfo has test case setup info.
-type SetupInfo struct {
+// createSession creates a new session and returns it.
+func createSession(region string) (*session.Session, error) {
+	if region == "" {
+		var err error
+		region, err = getRandomRegion()
+		if err != nil {
+			return nil, err
+		}
+		logging.Logger.Debugf("Creating session in region - %s", region)
+	}
+	session, err := session.NewSession(&awsgo.Config{
+		Region: awsgo.String(region)},
+	)
+	return session, err
+}
```

Comment:
NIT: since the namespace is shared across the entire package, this should probably be put in a different place or renamed to something specific to s3 tests.

Reply:
Done - added S3Test prefix where possible for consistency.